docs/build/circuit-construction.ipynb

```diff
@@ -0,0 +1,486 @@
+      "text/plain": [
+       "<Figure size 185.453x117.056 with 1 Axes>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit.library import HGate\n",
+    "\n",
+    "qc = QuantumCircuit(1)\n",
+    "qc.append(\n",
+    "    HGate(),  # New HGate instruction\n",
+    "    [0]       # Apply to qubit 0\n",
+    ")\n",
+    "qc.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bbdbef08-07c0-4fb0-ab3e-ed6efb7a3dc3",
+   "metadata": {},
+   "source": [
+    "To combine two circuits, use the [`compose`](/api/qiskit/qiskit.circuit.QuantumCircuit#compose) method. This accepts another [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit) and an optional list of qubit mappings.\n",
+    "\n",
+    "<Admonition type=\"note\">\n",
+    "    The [`compose`](/api/qiskit/qiskit.circuit.QuantumCircuit#compose) method returns a new circuit and does **not** mutate either circuit it acts on. To mutate the circuit you call [`compose`](/api/qiskit/qiskit.circuit.QuantumCircuit#compose) from, use the argument `inplace=True`.\n",
```

Comment:
```suggestion
    "    The [`compose`](/api/qiskit/qiskit.circuit.QuantumCircuit#compose) method returns a new circuit and does **not** mutate either circuit it acts on. To mutate the circuit, call [`compose`](/api/qiskit/qiskit.circuit.QuantumCircuit#compose) from and use the argument `inplace=True`.\n",
```

Reply:
@frankharkins - "call compose from and use the argument..." feels like something is missing.  Is the method "compose from"? Or are you calling it from somewhere? or...

Reply:
Its supposed to be:

> To mutate (the circuit you call `compose` from), use the argument...

I get how that could be difficult to parse but I'm not sure how to make it clearer. Maybe @javabster could help?

Reply:
I think it doesn't help that the name of the method is a verb and it doesn't render the `()`.

I think you could say "to mutate the circuit on which you are calling the `compose` method, use the argument..."

Reply:
That is better, thanks!

```suggestion
    "    The [`compose`](/api/qiskit/qiskit.circuit.QuantumCircuit#compose) method returns a new circuit and does **not** mutate either circuit it acts on. To mutate the circuit on which you're calling the [`compose`](/api/qiskit/qiskit.circuit.QuantumCircuit#compose) method, use the argument `inplace=True`.\n",
```